package.json

```diff
@@ -8,7 +8,7 @@
     "lint:fix": "tslint -p tsconfig.json --fix",
     "build": "tsc -p tsconfig.json",
     "watch:build": "tsc -p tsconfig.json --watch",
-    "start": "npm run build && node ./dist/src/server.js",
+    "start": "ts-node src/server.ts",
```

Comment:
Not against the change, but is there any particular reason for it?

Reply:
It's just to have a start command to run the server without the need of recompiling it each time.